<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RecoverPasswordDialog</class>
 <widget class="QDialog" name="RecoverPasswordDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Recover Password</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 #1b1f2a, stop:1 #10131a);
color: white;</string>
  </property>
  <widget class="QLabel" name="label_title">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>300</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent; color: white;</string>
   </property>
   <property name="text">
    <string>Password Recovery</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="email_input">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>300</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #2c2f3f; border: 1px solid #4C5C75; border-radius: 5px; padding: 5px; color: white;</string>
   </property>
   <property name="placeholderText">
    <string>Enter your email</string>
   </property>
  </widget>
  <widget class="QPushButton" name="send_code_button">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>140</y>
     <width>100</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #0078d7; color: white; border-radius: 5px; font-weight: bold;</string>
   </property>
   <property name="text">
    <string>Send Code</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="code_input">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>200</y>
     <width>300</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #2c2f3f; border: 1px solid #4C5C75; border-radius: 5px; padding: 5px; color: white;</string>
   </property>
   <property name="placeholderText">
    <string>Enter verification code</string>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="verify_code_button">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>260</y>
     <width>100</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #0078d7; color: white; border-radius: 5px; font-weight: bold;</string>
   </property>
   <property name="text">
    <string>Verify Code</string>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="new_password_input">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>320</y>
     <width>300</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #2c2f3f; border: 1px solid #4C5C75; border-radius: 5px; padding: 5px; color: white;</string>
   </property>
   <property name="placeholderText">
    <string>New password</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="confirm_password_input">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>380</y>
     <width>300</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #2c2f3f; border: 1px solid #4C5C75; border-radius: 5px; padding: 5px; color: white;</string>
   </property>
   <property name="placeholderText">
    <string>Confirm new password</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="reset_password_button">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>440</y>
     <width>100</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #0078d7; color: white; border-radius: 5px; font-weight: bold;</string>
   </property>
   <property name="text">
    <string>Reset Password</string>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>